index.d.ts

```diff
@@ -1341,6 +1347,85 @@ declare namespace Moleculer {
 	const CIRCUIT_CLOSE: string;
 	const CIRCUIT_HALF_OPEN: string;
 	const CIRCUIT_OPEN: string;
+
+	class Vorpal {
```

Comment:
@icebob Is it fair to suggest that moleculerjs/vorpal will now be maintained?  If so, it would probably be best to add these types there, cut a new release of that package, and then bump the package.json as part of this PR.  In that case, the new types would be available to both places.

Reply:
I don't say it will be maintained because the source code is sophisticated, it was really tough to solve the last vulnerabilities issue, as well. I don't think we are able to add any other changes to this code. Of course, a type definition file can be easier, but I think Wallacy's definition is just a part of the full vorpal types, right?

Reply:
I can put this on the moleculer/vorpal, also on molecular-repl, but i just put here because there's no previous type definition on those projects. This is a full vorpal definition by the way.

Reply:
tomorrow I want to release a new moleculer version. @shawnmcknight if you said that the ts definition is correct, I will merge it & publish and in the future we will able to move it to the relevant repo, as well.

Reply:
As a Typescript developer, this is a very straight forward definition anyway.